### PR TITLE
Add Aditya-mathur card info

### PR DESCRIPTION
**Pull request by AdityaMathur25**
This is my card please merge it

### CODE REVIEW

**Review by TenTraicion — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -114,6 +114,32 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+
+        <!-- ________ Aditya's card START ________  -->
+        <div class="card">
+          <p class="name">Aditya Mathur</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="_blank">@Aditya25062</a>
```
Comment (TenTraicion) [inline index.html:123]:
```suggestion
            <a href="https://www.twitter.com/Aditya25062" target="_blank">@Aditya25062</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
Kindly do the required changes and commit!

File: index.html (hunk 1 above)
Comment (AdityaMathur25) [reply index.html]: done please reverify it

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@AdityaMathur25 , Thanks for your Contribution. and Congrats on your first PR Merge in this project. Your Card will be live within ~2-3 minutes, which can be found: [Here](https://syknapse.github.io/Contribute-To-This-Project/)!

**Pull request merged**